Specs/DateTime/English/DateTimeModel.json

```diff
@@ -23962,5 +23962,206 @@
         }
       }
     ]
+  },
+  {
+    "Input": "I'll be back 7:56:30 am",
+    "Comment": "Taskmode changes resolution of past time resolution to datetime.",
```

Comment:
Same as previous task mode PR comments. If the behaviour is correct/desired in the default mode, no need for such comments.